packages/datadog-plugin-express/test/index.spec.js

```diff
@@ -856,6 +854,36 @@ describe('Plugin', () => {
           const app = express()
           const error = new Error('boom')
 
+          app.use((req, res, next) => next(error))
+          app.use((error, req, res, next) => res.status(500).send())
+
+          getPort().then(port => {
+            agent
+              .use(traces => {
+                const spans = sort(traces[0])
+
+                expect(spans[3]).to.have.property('error', 1)
+                expect(spans[3].meta).to.have.property('error.type', error.name)
+                expect(spans[3].meta).to.have.property('error.msg', error.message)
+                expect(spans[3].meta).to.have.property('error.stack', error.stack)
+              })
+              .then(done)
+              .catch(done)
+
+            appListener = app.listen(port, 'localhost', () => {
+              axios
+                .get(`http://localhost:${port}/user`, {
+                  validateStatus: status => status === 500
+                })
+                .catch(done)
+            })
+          })
+        })
+
+        it('should handle middleware exceptions', done => {
```

Comment:
What's the difference between this and the previous one.

Reply:
Throwing doesn't call `next`, so it's not the same code path to `throw error` or `next(error)`.

Reply:
Got it :+1: